src/main/scala/org/broadinstitute/clio/model/indexes.scala

```diff
@@ -1,18 +1,44 @@
 package org.broadinstitute.clio.model
 
+import java.time.OffsetDateTime
+
 case class ElasticsearchIndex(indexName: String, indexType: String, fields: Seq[ElasticsearchField])
 
 case class ElasticsearchField(fieldName: String, fieldType: Class[_])
 
 case class ReadGroup
 (
+  initiative: String,
```

Comment:
A combination of looking at the field order in the Picard dev database, and then guessing. Very open to a better order, but I don't think Elasticsearch nor Scala care.